### ISSUE TITLE
Lutris not capable of opening wine; Repository update for Arch Linux required

### ISSUE THREAD

**Issue by MephresGH** [lutris/lutris#4198]
I am currently facing issues with Lutris, as it is incapable of launching Wine, Winetricks, Wine settings and more, which leads to games not being able to launch via Wine, although I can run .exe files without any hassle via the command line. I tried re-installing everything, from Wine, Winetricks and Lutris itself, but nothing worked. I attempted it with both standard Wine and the Staging branch of it, but to no avail. Reboots were done after each change. 
Another issue seems to be that Lutris hasn't updated its repository for Arch, since I'm still stuck with 0.5.9.1.5 instead of 0.5.10. This might be the reason why as well.
I am on the newest version of EndeavourOS and every other tasks I do work with ease on it.

EDIT: I am now capable of running programs after having run Lutris via the console and the "-d" parameter, which is pretty odd. That part of the issue is done now, I'd say.

**Comment by HugoPlacer** [comment]
Same happens to me

**Comment by MephresGH** [comment]
> Same happens to me

Well, as I stated in my edit, it seems to fix itself after running it via the "-d" parameter in console. Another method could be to re-install Lutris and all its dependencies, including Wine and possibly even Proton, if required. However, the outdated package still is an issue.

**Comment by nephitejnf** [comment]
I have the same problem, but is looks like WINE is trying to run proton or steam wrappers.

```
WARNING  2022-04-07 23:38:19,099 [game.stop_game:556]:The game has run for a very short time, did it crash?
DEBUG    2022-04-07 23:38:21,907 [xrandr._get_vidmodes:14]:Retrieving video modes from XrandR
DEBUG    2022-04-07 23:38:27,182 [game.save:257]:Saving Total Annihilation (wine) with config ID total-annihilation-gog-1591760825
DEBUG    2022-04-07 23:38:27,182 [config.save:222]:Saving LutrisConfig(level=game, game_config_id=total-annihilation-gog-1591760825, runner=wine) config to /home/nephitejnf/.config/lutris/games/total-annihilation-gog-1591760825.yml
Traceback (most recent call last):
  File "/usr/lib/python3.10/site-packages/lutris/exceptions.py", line 57, in wrapper
    return function(*args, **kwargs)
  File "/usr/lib/python3.10/site-packages/lutris/game.py", line 420, in configure_game
    gameplay_info = self.get_gameplay_info()
  File "/usr/lib/python3.10/site-packages/lutris/game.py", line 398, in get_gameplay_info
    gameplay_info = self.runner.play()
  File "/usr/lib/python3.10/site-packages/lutris/runners/wine.py", line 825, in play
    launch_info = {"env": self.get_env(os_env=False)}
  File "/usr/lib/python3.10/site-packages/lutris/runners/wine.py", line 749, in get_env
    env = super(wine, self).get_env(False)
  File "/usr/lib/python3.10/site-packages/lutris/runners/runner.py", line 186, in get_env
    runtime_env = self.get_runtime_env()
  File "/usr/lib/python3.10/site-packages/lutris/runners/wine.py", line 789, in get_runtime_env
    for proton_path in get_proton_paths():
  File "/usr/lib/python3.10/site-packages/lutris/util/wine/wine.py", line 49, in get_proton_paths
    for path in _iter_proton_locations():
  File "/usr/lib/python3.10/site-packages/lutris/util/wine/wine.py", line 38, in _iter_proton_locations
    for path in [os.path.join(p, "common") for p in steam().get_steamapps_dirs()]:
  File "/usr/lib/python3.10/site-packages/lutris/runners/steam.py", line 243, in get_steamapps_dirs
    library_config = self.get_library_config()
  File "/usr/lib/python3.10/site-packages/lutris/runners/steam.py", line 149, in get_library_config
    return read_library_folders(self.steam_data_dir)
  File "/usr/lib/python3.10/site-packages/lutris/util/steam/config.py", line 161, in read_library_folders
    library["libraryfolders"].pop("contentstatsid")
KeyError: 'contentstatsid'
DEBUG    2022-04-07 23:38:34,326 [game.stop_game:554]:Total Annihilation (wine) has run for 0 seconds
WARNING  2022-04-07 23:38:34,326 [game.stop_game:556]:The game has run for a very short time, did it crash?
```

**Comment by MephresGH** [comment]
> I have the same problem, but is looks like WINE is trying to run proton or steam wrappers.
> 
> ```
> WARNING  2022-04-07 23:38:19,099 [game.stop_game:556]:The game has run for a very short time, did it crash?
> DEBUG    2022-04-07 23:38:21,907 [xrandr._get_vidmodes:14]:Retrieving video modes from XrandR
> DEBUG    2022-04-07 23:38:27,182 [game.save:257]:Saving Total Annihilation (wine) with config ID total-annihilation-gog-1591760825
> DEBUG    2022-04-07 23:38:27,182 [config.save:222]:Saving LutrisConfig(level=game, game_config_id=total-annihilation-gog-1591760825, runner=wine) config to /home/nephitejnf/.config/lutris/games/total-annihilation-gog-1591760825.yml
> Traceback (most recent call last):
>   File "/usr/lib/python3.10/site-packages/lutris/exceptions.py", line 57, in wrapper
>     return function(*args, **kwargs)
>   File "/usr/lib/python3.10/site-packages/lutris/game.py", line 420, in configure_game
>     gameplay_info = self.get_gameplay_info()
>   File "/usr/lib/python3.10/site-packages/lutris/game.py", line 398, in get_gameplay_info
>     gameplay_info = self.runner.play()
>   File "/usr/lib/python3.10/site-packages/lutris/runners/wine.py", line 825, in play
>     launch_info = {"env": self.get_env(os_env=False)}
>   File "/usr/lib/python3.10/site-packages/lutris/runners/wine.py", line 749, in get_env
>     env = super(wine, self).get_env(False)
>   File "/usr/lib/python3.10/site-packages/lutris/runners/runner.py", line 186, in get_env
>     runtime_env = self.get_runtime_env()
>   File "/usr/lib/python3.10/site-packages/lutris/runners/wine.py", line 789, in get_runtime_env
>     for proton_path in get_proton_paths():
>   File "/usr/lib/python3.10/site-packages/lutris/util/wine/wine.py", line 49, in get_proton_paths
>     for path in _iter_proton_locations():
>   File "/usr/lib/python3.10/site-packages/lutris/util/wine/wine.py", line 38, in _iter_proton_locations
>     for path in [os.path.join(p, "common") for p in steam().get_steamapps_dirs()]:
>   File "/usr/lib/python3.10/site-packages/lutris/runners/steam.py", line 243, in get_steamapps_dirs
>     library_config = self.get_library_config()
>   File "/usr/lib/python3.10/site-packages/lutris/runners/steam.py", line 149, in get_library_config
>     return read_library_folders(self.steam_data_dir)
>   File "/usr/lib/python3.10/site-packages/lutris/util/steam/config.py", line 161, in read_library_folders
>     library["libraryfolders"].pop("contentstatsid")
> KeyError: 'contentstatsid'
> DEBUG    2022-04-07 23:38:34,326 [game.stop_game:554]:Total Annihilation (wine) has run for 0 seconds
> WARNING  2022-04-07 23:38:34,326 [game.stop_game:556]:The game has run for a very short time, did it crash?
> ```

I'd highly recommend uninstalling all references to Lutris and Proton, clearing up any caches and possibly saved/leftover setting configurations and moving on from there. However, you should also look at the Wine-focused settings in Lutris and make sure no oddities are rearing their ugly heads.

**Comment by strycore** [comment]
> `library["libraryfolders"].pop("contentstatsid")`

Fixed in 0.5.10
